docs/contributing/style-guide.md

```diff
@@ -125,15 +171,28 @@ For a Closed Beta release, you'll need to exclude it from the nav and search eng
   ```
 1. Do _not_ add the doc to `sidebars.ts`.
 1. Publish the doc.
-1. When the feature is moved from Beta to GA, remove the Robots meta tag and Beta label, and add the doc to sidebars.ts.
 
+When the feature moves from Beta to GA, remove the Robots meta tag, remove the Beta label, and add the doc to sidebars.ts.
+
+### Open Beta
```

Comment:
Can you add a sentence that describes what an open beta is?

Reply:
Thanks, @jpipkin1 ! I'll create an Asana ticket for defining open/closed beta.